src/frontend/src/lib/components/icons/IconSearchClose.svelte

```diff
@@ -0,0 +1,27 @@
+<!-- source: DFINITY foundation -->
```

Comment:
this is not from Lucide? I had understood that we were using only from Lucide

Reply:
I explicitely asked Artem if I should export from Figma and he said yes. So assumed it was de facto a DFINITY foundation icon. Is that incorrect?